packages/@o3r-training/tools/package.json

```diff
@@ -0,0 +1,135 @@
+{
+  "name": "@o3r-training/tools",
+  "version": "0.0.0-placeholder",
+  "private": true,
+  "description": "Otter training tools",
+  "module": "dist/src/public_api.js",
+  "esm2015": "dist/esm2015/public_api.js",
+  "esm2020": "dist/src/public_api.js",
+  "typings": "dist/src/public_api.d.ts",
+  "keywords": [
+    "utilities",
+    "otter-training"
+  ],
+  "scripts": {
+    "nx": "nx",
+    "ng": "yarn nx",
+    "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q --strip-leading-paths",
+    "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q --strip-leading-paths",
+    "build:esm2020": "tsc -b tsconfig.build.json --pretty",
+    "build": "yarn nx build training-tools",
+    "postbuild": "yarn cpy 'package.json' dist && patch-package-json-main",
+    "prepare:publish": "prepare-publish ./dist",
+    "build:source": "yarn build:esm2020 && yarn build:cjs && yarn build:esm2015",
```

Comment:
Do you really need to transpile in so many mode?
ESM build would not be enough?